pages/common/ccache.md

```diff
@@ -0,0 +1,24 @@
+# ccache
+
+> C/C++ compiler cache.
+> More information: <https://linux.die.net/man/1/ccache>.
```

Comment:
```suggestion
> More information: <https://ccache.dev/manual/4.9.html#_command_line_options>.
```
For man page links, we prefer to use manned.org (would be https://manned.org/ccache in this case) as a fallback, but the priority is the official documentation. From the style guide:
> ## More information links
> 
> - On the `More information` link line, we prefer linking to the author's provided documentation of the command line reference or the man page. When not available, use <https://manned.org> as the default fallback for all platforms (except `osx` and BSD platforms other than FreeBSD). Alternatively, you can link to the author's website or a tutorial page if the command doesn't have a documentation page.